easybuild/easyconfigs/m/MDSplus/mdsplus-7.132.0_paths.patch

```diff
@@ -0,0 +1,239 @@
+diff -Nru mdsplus-java-7.131.6-orig/mdsplus/java/jdispatcher/jDispatcherIp.template mdsplus-java-7.131.6/mdsplus/java/jdispatcher/jDispatcherIp.template
```

Comment:
Please add description and author/date info to the patch (and update checksums)

Reply:
Thanks @akesandgren, I've added a very brief description to the patch and updated as you suggested.